### PR TITLE
Add maven publish plugin to auth modules

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add maven publish plugin to auth modules.

#### WHY

So they are published to maven in the next release for early API feedback.

#### HOW

- Uncomment line to apply maven publish plugin;
- Add missing configuration files;
- Unrelated: add [resourcePrefix](https://github.com/google/horologist/issues/892);

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [N/A] Update metalava's signature text files

### CODE REVIEW

File: auth-composables/build.gradle.kts [hunk 1]
```diff
@@ -75,6 +75,8 @@ android {
         textReport = true
     }
 
+    resourcePrefix = "horologist_"
```
Comment (yschimke) [inline auth-composables/build.gradle.kts:78]: Why this change?

**Review by yschimke — APPROVED** [review]

File: auth-composables/build.gradle.kts (hunk 1 above)
Comment (luizgrp) [reply auth-composables/build.gradle.kts:78]: https://github.com/google/horologist/issues/892
